deebotozmo/cli.py

```diff
@@ -0,0 +1,439 @@
+    await run_with_login(SetWaterLevel(level))
+
+
+@cli.command(help="Returns to charger")
+@coro
+async def charge() -> None:
+    """Click subcommand that returns to charger."""
+    await run_with_login(Charge())
+
+
+@cli.command(help="Pause the robot")
+@coro
+async def pause() -> None:
+    """Click subcommand that pauses the clean."""
+    await run_with_login(CleanPause())
+
+
+@cli.command(help="Resume the robot")
+@coro
+async def resume() -> None:
+    """Click subcommand that resumes the clean."""
+    await run_with_login(CleanResume())
+
+
+@cli.command(name="getcleanlogs", help="Get Clean Logs")
+@coro
+async def get_clean_logs() -> None:
+    """Click subcommand that returns clean logs."""
+    vacbot = DoLogin()
+    await vacbot.run()
+
+    lock = asyncio.Lock()
+    await lock.acquire()
+
+    async def on_clean_event(event: CleanLogEvent) -> None:
+        print(json.dumps(asdict(event)))
+        lock.release()
+
+    listener = vacbot.bot.events.clean_logs.subscribe(on_clean_event)
+    await lock.acquire()
+    vacbot.bot.events.clean_logs.unsubscribe(listener)
+
+    await vacbot.goodbye()
+
+
+@cli.command(help="Get robot statuses [Status,Battery,FanSpeed,WaterLevel]")
+@coro
+async def statuses() -> None:
+    """Click subcommand that returns the robot status."""
+    vacbot = DoLogin()
+    await vacbot.run()
+    lock = asyncio.Lock()
+
+    print(f"Vacuum State: {str(vacbot.bot.status.state).rsplit('.', maxsplit=1)[-1]}")
+
+    await lock.acquire()
+
+    async def on_battery(event: BatteryEvent) -> None:
+        print("Battery: " + str(event.value) + "%")
+        lock.release()
+
+    battery_listener = vacbot.bot.events.battery.subscribe(on_battery)
+    await lock.acquire()
+    vacbot.bot.events.battery.unsubscribe(battery_listener)
+
+    async def on_fan_event(event: FanSpeedEvent) -> None:
+        print("Fan Speed: " + str(event.speed))
+        lock.release()
+
+    fan_speed_listener = vacbot.bot.events.fan_speed.subscribe(on_fan_event)
+    await lock.acquire()
+    vacbot.bot.events.fan_speed.unsubscribe(fan_speed_listener)
+
+    async def on_water_level(event: WaterInfoEvent) -> None:
+        print("Water Level: " + str(event.amount))
+        lock.release()
+
+    water_level_listener = vacbot.bot.events.water_info.subscribe(on_water_level)
+    await lock.acquire()
+    vacbot.bot.events.water_info.unsubscribe(water_level_listener)
+
+    await vacbot.goodbye()
+
+
+@cli.command(help="Get stats")
+@coro
+async def stats() -> None:
+    """Click subcommand that returns bot stats."""
+    vacbot = DoLogin()
+    await vacbot.run()
+
+    lock = asyncio.Lock()
+    await lock.acquire()
+
+    async def on_stats_event(event: StatsEvent) -> None:
+        print("Stats Cid: " + str(event.clean_id))
+        print("Stats Area: " + str(event.area))
+        if isinstance(event.time, int):
+            print("Stats Time: " + str(int(event.time / 60)) + " minutes")
```

Comment:
Please use f strings everywhere so it's consist through the whole code.
See Line 317